src/sdp/str.c

```diff
@@ -22,6 +22,7 @@ const char sdp_media_text[]    = "text";      /**< Media type 'text'    */
 
 const char sdp_proto_rtpavp[]  = "RTP/AVP";   /**< RTP Profile          */
 const char sdp_proto_rtpsavp[] = "RTP/SAVP";  /**< Secure RTP Profile   */
+const char sdp_proto_udp[]     = "udp";       /**< Direct use of UDP    */
```

Comment:
is this variable used in the patch ?

Reply:
No.  I added it for consistency.  Should I remove it?

Reply:
in general a symbol should be used inside the library or exported in public API.
if the symbol is unused it should be removed

Reply:
I made the requested change.  Your software, your rules :)